Library/Formula/jdnssec-tools.rb

```diff
@@ -0,0 +1,21 @@
+class JdnssecTools < Formula
+  desc "Java command-line tools for DNSSEC"
+  homepage "http://www.verisignlabs.com/jdnssec-tools/"
+  url "http://www.verisignlabs.com/dnssec-tools/packages/jdnssec-tools-0.12.tar.gz"
+  sha256 "3fa0bfe062051d9048c5186adbb04d8f5e176e50da3cdcf9b0d97e1c558efc22"
+
+  depends_on :java
+
+  def install
+    inreplace Dir["bin/*"] do |s|
+      s.gsub! /basedir=.*/, "basedir=#{libexec}"
+    end
+    bin.install Dir["bin/*"]
```

Comment:
Clarification. A lot of JAVA stuff roots itself in the directory, so if you move executables from `libexec` (for example) directly to `bin` and don't move the `basedir`/`home`/etc you get failure messages on run. Just wanted to check that wasn't the case here.